functions/system.bash

```diff
@@ -213,6 +213,7 @@ misc_system_settings() {
   echo "OK"
 }
 
+
```

Comment:
Why the random newline addition?

Reply:
(*puzzled*) see below how the code looks on *my* machine, the extra newline is to separate variable declarations from code. But I removed it now, too, to be in line with the rest of the code.

  echo "OK"
}

## change system swap size dependent on free space on /
## swap on SD (/var/swap per default) only used after ZRAM swap full if that exists
##
##    change_swapsize(int size in MB)
##
change_swapsize() {
  local totalMemory


  if ! is_raspbian; then return 0; fi

Reply:
Looks better, sometimes GitHub makes it really hard to tell what is going on from the diff. Either way it will be standardized once I get around to doing the backup functions.